types/jest/jest-tests.ts

```diff
@@ -186,11 +182,7 @@ fit.skip("name", () => {}, 9001);
 fit.skip("name", async () => {}, 9001);
 fit.skip("name", (callback: jest.DoneCallback) => {}, 9001);
 
-fit.todo("name", () => {});
-fit.todo("name", async () => {});
-fit.todo("name", () => {}, 9001);
-fit.todo("name", async () => {}, 9001);
-fit.todo("name", (callback: jest.DoneCallback) => {}, 9001);
+fit.todo("name");
```

Comment:
It seems `fit.todo`, `xit.todo` and `xtest.todo` are not valid function. I don't remove them because I just wanted to fix the valid `.todo` function. If you want them to be removed, I'm ok to do so